app/views/supervisor/dashboard/index.html.slim

```diff
@@ -1,11 +1,26 @@
 h2 Supervisor Dashboard
-p Alpha Version
+h5 Alpha Version
+
+div.dashboard-box
+  section.team-title
+    h3 Overview
+    h5 Under Construction
+  section.dashboard
+    p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
+    p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
+    /this is where all the aggregated messages and info should go
+    /= flash[:notice]
 
 div.clearfix
   - supervised_teams.each do |t|
     div.dashboard-box
       section.team-title
         h3 = link_to "Team " + t.name, t
+        div = render "#{ t.performance.score }"
+        - if t.performance.score == :orange
+          - flash.notice = "Team #{t.name} needs your attention"
```

Comment:
It would. For now. Handling of the messaging will be in the next iteration. We probably are going to use the flash notice for a general notice that there are teams who need attention, and place the team-related message in the overview box. We need feedback from the supervisors to see if the team-specific message adds value at all, now that we have icons.